<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SchedulePage</class>
 <widget class="QDialog" name="SchedulePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>842</width>
    <height>643</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>400</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose the new appointment date that suits you:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Manage your appointments:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>320</y>
     <width>361</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter the date of the appointment you want to cancel:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>140</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Current appointments:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>50</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Appointment Scheduling Page</string>
   </property>
  </widget>
  <widget class="QComboBox" name="AppDateComboBox">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>401</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>170</y>
     <width>341</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="AppDateComboBox_2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>320</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="updateappointmentbutton">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>401</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelappointmentbutton">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>330</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Back5">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>500</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
